tests/integration/cloudformation/api/test_changesets.py

```diff
@@ -328,7 +328,7 @@ def test_describe_change_set_nonexisting(snapshot, aws_client):
     snapshot.match("exception", ex.value)
 
 
-@pytest.mark.xfail(reason="fails because of the properties mutation in the result_handler")
+@pytest.mark.skip(reason="fails because of the properties mutation in the result_handler")
```

Comment:
don't see a point in waiting 5min for all the timeouts here, so this should speed up the tests a bit :sweat_smile: